### DISCUSSION TITLE
dotnet restore sometimes ignores http-cache

### DISCUSSION THREAD

**Discussion by asv7c2** [NuGet/Home#11042] [Q&A] [answered]
If delete some package from global package and then run dotnet restore it sometimes ignore package downloaded in http-cache and download package again. Is this correct behavior?

**Comment by asv7c2** [comment]
Okay, looks like it by design: http-cache has 30 minutes expiration. But why it has expiration? Package can be changed?

**Comment by nkolev92** [comment, marked as answer]
All caching is done at 30 min, there's an issue about changing that https://github.com/NuGet/Home/issues/3390
